tests/unittests/OperatorTest.cpp

```diff
@@ -1025,6 +1025,42 @@ TEST_P(OperatorTest, ReluSimple_Float16) {
   testReluSimple<float16_t>(bindings_, mod_, F_, EE_, ElemKind::Float16Ty);
 }
 
+/// Helper to test ReluSimple using \p DTy.
+template <typename DataType>
+static void testPReluSimple(glow::PlaceholderBindings &bindings,
+                            glow::Module &mod, glow::Function *F,
+                            glow::ExecutionEngine &EE, ElemKind DTy) {
+  auto *in = mod.createPlaceholder(DTy, {7}, "in", false);
+  auto *slope = mod.createPlaceholder(DTy, {7}, "slope", false);
+  auto *prelu = F->createPRELU("prelu", in, slope);
+  auto *save = F->createSave("prelu", prelu);
+  auto *result = bindings.allocate(save->getPlaceholder());
+
+  bindings.allocate(in)->getHandle<DataType>() = {0, -1, -2, -3, 4, 5, 6};
+  bindings.allocate(slope)->getHandle<DataType>() = {1, 1, 1, 1, 1, 1, 1};
+
+  EE.compile(CompilationMode::Infer, F);
+  EE.run(bindings);
+
+  auto resultH = result->getHandle<DataType>();
+  auto inH = bindings.get(in)->getHandle<DataType>();
+
+  for (size_t i = 0; i < 7; i++) {
+    EXPECT_EQ(resultH.raw(i), inH.raw(i));
```

Comment:
it may feel like it's always slope * input, can you change slope so that we better exercise that it's x for positive and slope*x for negative.